src/go/k8s/pkg/console/deployment.go

```diff
@@ -244,7 +244,7 @@ func (d *Deployment) syncSchemaRegistrySecret() (map[string][]byte, error) {
 	// Only write CA cert if not using DefaultCaFilePath
 	ca := &SecretTLSCa{
 		NodeSecretRef:  d.clusterobj.SchemaRegistryAPITLS().TLS.NodeSecretRef,
-		UsePublicCerts: UsePublicCerts,
+		UsePublicCerts: !d.consoleobj.Spec.SchemaRegistry.UseSchemaRegistryCA,
```

Comment:
Quick question, can any of these items be nil? I suspect not but just asking if there are any getters etc..

Reply:
No. The only pointer is `consoleobj` which is always non nil since the controller reconciles the consoleobj.